client/reader/stream/index.jsx

```diff
@@ -568,6 +591,7 @@ export default connect(
 	( state, { streamKey, recsStreamKey } ) => {
 		const stream = getStream( state, streamKey );
 		const selectedPost = getPostByKey( state, stream.selected );
+		const streamKeySuffix = streamKey?.substring( streamKey.indexOf( ':' ) + 1 );
```

Comment:
might need a null check on streamKey.indexOf
```suggestion
		const streamKeySuffix = streamKey?.substring( streamKey?.indexOf( ':' ) + 1 );
```